.github/workflows/i18n_server_list_update.yaml

```diff
@@ -0,0 +1,37 @@
+name: i18n Updater
+# This Workflow will periodically go in:
+# 1. Update extras.xliff with any new servers names Mullvad may have added
+# 2. File a pr with the changes to extras.xliff
+
+on:
+  schedule:
+    - cron: "0 7 * * *" # Every Day 7:00 UTC
```

Comment:
Maybe too much? LMK.

Reply:
Definitely too much, once a week is probably enough…